docs/LoaderDriverInterface.md

```diff
@@ -765,14 +748,19 @@ The following rules apply:
 * If the driver is unaware of any entrypoint with the name `pName`, it **must**
   return `NULL`.
 
-This support is optional and should not be considered a requirement.
-This is only required if a driver intends to support some functionality not
-directly supported by a significant population of loaders in the public.
+If a driver intends to support functions that take VkPhysicalDevice as the
+dispatchable parameter, then the driver should support
+`vk_icdGetPhysicalDeviceProcAddr`. This is because if these functions aren't
+known to the loader, such as they are from unreleased extensions or because
```

Comment:
"such as they are from" -> "such as those from"

I offer this poor and humble suggestion based on my meagre understandings of English, _"such as they are"_.